todo_app/app.py

```diff
@@ -8,4 +9,40 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = get_items()
+    completed_items = [item for item in items if item['complete']]
+    uncompleted_items = [item for item in items if not item['complete']]
+
+    return render_template('index.html', completed_items=completed_items, uncompleted_items=uncompleted_items)
+
+@app.route('/task', methods=['POST'])
+def add_new_task():
+    title = request.form.get('new_task_title')
+    description = request.form.get('new_task_description')
+    
+    if title:
```

Comment:
Nice simple validation here. If we are doing this though, we might as well throw a `required` attribute on the title input so that we can give the users very clear obvious feedback if they accidentally try to submit something blank

Reply:
Nice didn't know about that - thanks!